features/support/sharedSteps.js

```diff
@@ -0,0 +1,29 @@
+const pactum = require('pactum');
```

Comment:
What makes a step suitable for the `sharedSteps` file over the `steps` file? Would it make sense to combine the two files since some of the steps in `steps.js` are going to be used for all the features?

Reply:
We took steps.js directly from the pactum source examples, would like to keep it separated in case of pactum updates in the future.